salt/returners/pgjsonb.py

```diff
@@ -59,6 +59,22 @@
     alternative.pgjsonb.ssl_cert: '/etc/pki/mysql/certs/localhost.crt'
     alternative.pgjsonb.ssl_key: '/etc/pki/mysql/certs/localhost.key'
 
+Should you wish the returner data to be cleaned out every so often, set
+`keep_jobs` to the number of hours for the jobs to live in the tables.
```

Comment:
I know this is very picky, but these instances using single back-ticks throughout these new docs need to be using double back-ticks in order for our docs to display these terms as `code-format`. (Markdown uses single or double, RST uses double-only.)